src/doc_scan_service/session/create/filters/required.supplementary.document.builder.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const RequiredSupplementaryDocument = require('./required.supplementary.document');
+const Validation = require('../../../../yoti_common/validation');
+const Objective = require('./objective/objective');
+
+class RequiredSupplementaryDocumentBuilder {
+  /**
+   * @param {string[]} countryCodes
+   *
+   * @returns {this}
+   */
+  withCountryCodes(countryCodes) {
```

Comment:
Do we have the singular WithCountryCode & WithDocumentType for DocumentFilters? If so, I think we should add here too

Reply:
This hasn't been added to Java, so perhaps we should leave this out for now?